app/assets/javascripts/select-kit/addon/components/bulk-select-topics-dropdown.js

```diff
@@ -1,9 +1,32 @@
 import { action } from "@ember/object";
 import { service } from "@ember/service";
-import BulkTopicActions from "discourse/components/modal/bulk-topic-actions";
+import BulkTopicActions, {
+  _addBulkDropdownAction,
+} from "discourse/components/modal/bulk-topic-actions";
 import i18n from "discourse-common/helpers/i18n";
 import DropdownSelectBoxComponent from "select-kit/components/dropdown-select-box";
 
+const _customButtons = [];
+const _customOnSelection = {};
+
+export function _addBulkDropdownButton(opts) {
```

Comment:
Same comment about this being a weird `_`, should just be `addBulkDropdownButton` when exporting